docs/design/sno-board.md

```diff
@@ -0,0 +1,50 @@
+# SNOBoard
+
+## OVERVIEW
+
+The SNOBoard is a GUI for Storage Node Operators to get more information about their storage nodes performance as it pertains to the network or individual satellites.
+
+Wireframes:
+https://www.figma.com/file/IlY5UNF94rEpCLGR6BeaOhXX/Local-Storage-Node-Dashboard-Low-Fidelity-Mock?node-id=7%3A520 
+
+## GOALS
+
+- Give SNOs a pretty graphical user interface they can look at to get metrics on:
+	- Node(s) performance per satellite
+	- Storage space
+  	- Bandwidth
+	- Audit information
+	- Uptime information
+  	- Current Status
+	- SN version currently running
+	- DQ’ed status
+	- Containment status
+- Satellite notifications
+	- Notifications center on the SNOBoard
+	- Email notification opt-in
+- Expected payout per satellite, per node
```

Comment:
Should we also give info about the held amount in the escrow?

Reply:
Which would require node age to calculate it. Node age per satellite.